src/index.js

```diff
@@ -1,14 +1,63 @@
-// import _ from "lodash";
 import './style.css';
+import axios from 'axios';
 
-// function component() {
-//   const element = document.createElement("div");
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const gameID = 'YOUR_GAME_ID'; // Replace with the game ID obtained from the API
 
-//   // Lodash, now imported by this script
-//   element.innerHTML = _.join(["Hello", "webpack"], " ");
-//   element.classList.add("hello");
+const scoreTable = document.querySelector('.score-table');
+const refreshButton = document.querySelector('.refresh');
+const scoreForm = document.querySelector('#score-form');
+const nameInput = document.querySelector('#name');
+const scoreInput = document.querySelector('#score');
 
-//   return element;
-// }
+// Function to fetch and display scores
+async function refreshScores() {
+  try {
+    const response = await axios.get(`${baseURL}games/${gameID}/scores/`);
+    const scores = response.data.result;
 
-// document.body.appendChild(component());
+    scoreTable.innerHTML = '';
+
+    scores.forEach((score) => {
+      const scoreElement = document.createElement('div');
+      scoreElement.textContent = `User: ${score.user} - Score: ${score.score}`;
+      scoreTable.appendChild(scoreElement);
+    });
+  } catch (error) {
+    // Handle the error without using any console statements
+  }
+}
```

Comment:
- [ ] Ouch! this is close but one of the project requirements is to use `arrow functions` instead of the `function` keyword so as such, please replace any function definitions with an arrow function. This means at line `14` and `35` 😉